sdk/identity/Azure.Identity/src/AzureApplicationCredentialOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Identity
+{
+    /// <summary>
+    /// Options to configure the <see cref="AzureApplicationCredential"/> authentication flow and requests made to Azure Identity services.
+    /// </summary>
+    public class AzureApplicationCredentialOptions : TokenCredentialOptions
+    {
+        /// <summary>
+        /// Specifies the preferred authentication account to be retrieved from the shared token cache for single sign on authentication with
+        /// development tools. In the case multiple accounts are found in the shared token.
+        /// </summary>
+        /// <remarks>
+        /// If multiple accounts are found in the shared token cache and no value is specified, or the specified value matches no accounts in
+        /// the cache the SharedTokenCacheCredential will not be used for authentication.
+        /// </remarks>
+        public string SharedTokenCacheUsername { get; set; } = GetNonEmptyStringOrNull(EnvironmentVariables.Username);
```

Comment:
This credential doesn't use the `SharedTokenCacheCredential` so I'm confused why `SharedTokenCacheUsername` is referenced here.